### PR TITLE
Korean translation update

### PR DESCRIPTION
**Pull request by jungsomyeong**
Korean translation update

### CODE REVIEW

**Comment by MilosKozak** [comment]
duplicated translation on line 1042

File: lib/language.js [hunk 1]
```diff
@@ -2016,6 +2039,7 @@ function init() {
       ,ru: 'Стереть запись'
       ,sk: 'Zmazať záznam'
       ,nl: 'Verwijder invoer'
+      ,ko: '삭제'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2737522851/issues/source?bid=2452379&fileBranchId=3352843#l2042)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2737522851/issues/source?bid=2452379&fileBranchId=3352843#l2042)

File: lib/language.js [hunk 2]
```diff
@@ -2016,6 +2038,7 @@ function init() {
...
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2737942153/issues/source?bid=2452379&fileBranchId=3352843#l2041)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2737942153/issues/source?bid=2452379&fileBranchId=3352843#l2041)

**Comment by jungsomyeong** [comment]
I will new request so I close this request.

**Pull request closed without merging**